pkg/eval/chdir_test.go

```diff
@@ -73,17 +73,13 @@ func TestChdirError(t *testing.T) {
 	}
 }
 
-// Saves the current working directory, and returns a function for returning to
-// it.
+// Save the current working directory and return a function for returning to it.
```

Comment:
There is no need to change this; using third-person singular for unexported functions is the convention in the codebase.

Reply:
Yes, that is the case. Reverted. Having said that it's not obvious the passive voice implied by third-person singular is preferable. Not to mention the comma is superfluous.